pypesto/problem.py

```diff
@@ -95,7 +98,6 @@ def __init__(
         x_fixed_indices: Optional[SupportsIntIterableOrValue] = None,
         x_fixed_vals: Optional[SupportsFloatIterableOrValue] = None,
         x_guesses: Optional[Iterable[float]] = None,
-        startpoint_method: Optional[Callable] = None,
```

Comment:
I think it makes sense to have the startpoint outside of `Problem`, as it's rather a part of the algorithm for solving, than of the problem itself.